### PR TITLE
feat: Introduce Rights to Use model

### PR DESCRIPTION
**Pull request by aronhoyer**
Resolves [AB#86674](https://dev.azure.com/EquinorASA/d02c350d-092d-41bf-a812-f05d5d8ad1b0/_workitems/edit/86674)

### CODE REVIEW

**Comment by wenche** [comment]
What is description in this context?

**Comment by aronhoyer** [comment]
some of the rtu assets seem to have some further explanations on access control in the description. don't know if it's something we want to use, but thought i'd add it for posterity

**Comment by aronhoyer** [comment]
oh description should probably be a block type, huh? 👀

**Comment by wenche** [comment]
> oh description should probably be a block type, huh? 👀

Yes, if it is a rich text editor in Collibra

**Comment by wenche** [comment]
I think we should rather use @portabletext/types instead of creating our own?

**Comment by aronhoyer** [comment]
no idea that existed 😂 thanks!

**Comment by wenche** [comment]
> no idea that existed 😂 thanks!

We use that in the data-marketplace repo :)

File: index.ts [hunk 1]
```diff
@@ -1,2 +1,6 @@
 export type { Asset } from "./types/Asset"
 export type { User } from "./types/User"
+export type { Block } from "./types/Block"
```
Comment (wenche) [inline index.ts]: Shouldn't this be removed as well? A little bit hard to see in the files

**Review by wenche — COMMENTED** [review]
Should we remove the old homemade types as well

**Review by wenche — APPROVED** [review]
LGTM 👍

**Pull request merged**